src/query/expression/src/values.rs

```diff
@@ -1846,6 +1846,75 @@ impl ColumnBuilder {
         }
     }
 
+    pub fn push_repeat(&mut self, item: ScalarRef, n: usize) {
```

Comment:
Where to put unit tests?

Reply:
expression/tests

Reply:
so we can replace `ColumnBuilder::repeat`  with

```
let mut builder = ColumnBuilder::with_capacity(...);
builder.push_repeat(item, n);
```